node/src/service/p2p/bootstrap.rs

```diff
@@ -61,46 +104,166 @@ impl BootstrapConfig {
 pub(crate) async fn bootstrap(
     mut swarm: Swarm<BootstrapBehaviour>,
     addr: Multiaddr,
+    bootstrap_addresses: Vec<Multiaddr>,
 ) -> Result<(), P2PError> {
     info!("Starting P2P bootstrap node at {addr}");
     swarm.listen_on(addr)?;
+    for addr in bootstrap_addresses {
+        info!("Dialing {addr}");
+        swarm.dial(addr)?;
+    }
```

Comment:
What if one of the addressess fail to dial? I don't think we should fail the entire bootstrap if one of the nodes is unreachable.